cli/azd/cmd/root.go

```diff
@@ -82,6 +82,12 @@ func NewRootCmd(staticHelp bool, middlewareChain []*actions.MiddlewareRegistrati
 					"no-prompt",
 					false,
 					"Accepts the default value instead of prompting, or it fails if there is no default.")
+			rootCmd.PersistentFlags().
+				BoolVar(
+					&opts.Documentation,
+					"docs",
+					false,
+					"Go to the official documentation for the executed command.")
```

Comment:
Maybe we need a little word-smithing here.  I feel like "the executed command." is a little confusing in this context, since the command never gets executed.  It's also unclear what "Go to the official documentation" would do.

Maybe something like: "Open the documentation for this command in your default web browser"?

I wonder if @Austinauth or @gkulin have other ideas around phrasing.